doc/quickstart.md

```diff
@@ -0,0 +1,208 @@
+
+ Python is an **interpreter** language, **dynamically typed** and **garbage-collected**.
+
+ ### Python object
+
+-   is created by the Python interpreter when `object.__new__()` is invoked (e.g. as a result of an expression).
+-   can be either mutable or immutable, but its type never changes.
+-   resides in memory and has a **reference count**.
+-   is accessed through one or more Python variables.
+-   is destroyed by the garbage collector when its reference count drops to zero.
+
+For more details about Python object, see [this](https://docs.python.org/3/tutorial/classes.html).
+
+ ### Python variable
+
+-   is a reference to a Python object in memory.
+
+-   is created with an assignment operation `x = expr`:
+    -   if the variable `x` already exists, the interpreter reduces the reference count of its object
+    -   a new variable `x` is created, which references the value of expr.
+
+-   can be destroyed with the command del `x`.
+
+For more details about Python variable, see [this](https://www.w3schools.com/python/python_variables.asp).
+
+ ## Static typed languages
+	
+A language is statically-typed if the type of a variable is known at compile-time instead of at run-time. Common examples of statically-
+typed languages include Java, C, C++, FORTRAN, Pascal and Scala. See [this](https://en.wikipedia.org/wiki/Type_system#:~:text=In%20programming%20languages%2C%20a%20type,%2C%20expressions%2C%20functions%20or%20modules.) and [this](https://android.jlelse.eu/magic-lies-here-statically-typed-vs-dynamically-typed-languages-d151c7f95e2b#:~:text=Static%20typed%20languages,%2C%20FORTRAN%2C%20Pascal%20and%20Scala.) for more details.
+
+ ## Installation (see [README](https://github.com/pyccel/pyccel/blob/master/README.rst) file)
+
+ ## Command line usage
+
+-   Open a terminal app, iterm or terminal for MacOs, terminal for Linux.
+
+-   After the installation, type `pyccel`, the usage should be shown, then all good.
+
+-   Create a Python file that contains simple lines of code to see what will happen.
+    -   To create the file `touch file_name.py`.
+    -   Use your favorite text editor to fill the file with some lines of code or just type `echo 'print("hello, world!\n")' > file_name.py` for a quick test, `cat file_name.py` to make sure that your lines in the file.
+    -   To generate the C/Fortran code form your Python code, type `pyccel file_name.py` or `pyccel file_name.py --language fortran` to generate Fortran code, and `pyccel file_name.py --language c` to generate C code.
+    -   No problems ?, So you should discover `__pyccel__` the directory that contains your generated code and some other stuff.
+
+E.g (using `@types` decorator and a recursive function with a typed return), To specify the types of the function arguments and its return, we need to import the `@types` decorator from pyccel.decorators (as you can see in the first line) and then specify the types for each function using this format:
```

Comment:
(as you can see in the first line) 
I don't think that it is clear which line you are talking about